sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -369,9 +366,8 @@ export type MergeCertificateOptions = coreHttp.OperationOptions;
 
 // @public
 export interface ParsedKeyVaultCertificatesIdentifier {
-    collection: KeyVaultCertificatesIdentifierCollectionName;
-    id: string;
     name: string;
+    sourceId: string;
```

Comment:
Copying my feedback from https://github.com/Azure/azure-sdk-for-js/pull/9642#issuecomment-654382318 for reference

>  I would recommend changing the name in the output for this id to make it clear that it was the input id. Otherwise, when I look at a type called Parsed.... I expect it to contain the parsed bits. A reference to the original should be named that it is easily at a glance differentiable from the others.

Do we have precedence in any other methods (not necessarily Azure SDK related) in any of the languages where input is part of the output as well? Maybe we can get inspired from that?

Reply:
In this case I don't mind which approach we take, both `id` and `sourceId` seem valid. I'll leave it as `sourceId` for now.